src/test/java/org/broadinstitute/hellbender/utils/nio/PathLineIteratorUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PathLineIteratorUnitTest extends GATKBaseTest {
+
+  @Test
```

Comment:
Indentation looks weird in this file compare to others...

Reply:
Switched to 4-spaces indent.